packages/salesforcedx-vscode-apex/test/vscode-integration/apexLspJarContents.test.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+// tslint:disable:no-unused-expression
+
+import { expect } from 'chai';
+import * as path from 'path';
+import * as shell from 'shelljs';
+
+describe('Apex LSP Jar Contents', () => {
+
+  it('should not contain perm-guarded apex classes', () => {
```

Comment:
nit: can we use `it.each` here instead ?

Reply:
ehhhh.. I dunno that we should depend on a module that is infrequently used and hasn't been updated in a few years.

Reply:
I wasn't referring to the 3rd party mod. I was hoping our test runner/framework would have something similar to https://jestjs.io/docs/api#testeachtablename-fn-timeout

Reply:
ah...I'm not aware of anything in mocha that is equalivant.  Simplest thing might be something like
```
const clsNames = ['AuraException.cls', ...];
const checkClsName = (className :string, stdout: string) => {
  expect(stdout.includes(className).to.be.false;
}
clsName.forEach((name) => {
  checkClsName(name);
});
```

Reply:
I generally try to avoid loops in tests. Ideally, I would simplify this test into multiple tests so each tests one exact thing (and one class in this case). But felt it was overkill in this scenario.